capstone-ui/src/components/Search/Search.jsx

```diff
@@ -33,48 +32,52 @@ export default function Search({ username, token }) {
   };
 
   const searchSongs = async (e) => {
-    if (searchInput != "") {
-      let temp = searchInput;
-      if (searchInput === "") {
-        temp = searchInputValue;
-      }
-      setSearchInputValue(temp);
-
-      setProfileResults([]);
-
-      e.preventDefault();
-      if (!isSongResults) {
-        setOffset(0);
-      }
-
-      let oldResults = songResults;
-      const { data } = await axios
-        .get(
-          "https://api.spotify.com/v1/search",
-          {
-            params: {
-              q: temp,
-              type: "track",
-              include_external: "audio",
-              limit: 5,
-              offset: offset,
-            },
-          },
-          {
-            headers: {
-              Authorization: `Bearer ${token}`,
-            },
-          }
-        )
-        .catch((error) => {});
-
-      let newResults = data.tracks.items;
-      let addedResults = oldResults.concat(newResults);
-      setSongResults(addedResults);
-      setIsSongResults(true);
-      setOffset((previousValue) => previousValue + 5);
-      setSearchInput("");
+    let oldSearchResults;
+    let trueSearchValue = searchInput;
+    if (searchInput == "") {
+      trueSearchValue = searchInputValue;
+      oldSearchResults = songResults;
+    } else {
+      oldSearchResults = [];
     }
```

Comment:
suggestion on rewriting this as:
const oldSearchResults = searchInput == "" ? songResults : [];
const trueSearchValue = searchInput == ""  ? searchInputValue ? "";

Reply:
I fixed this in my Search PR just now so when I merge I will make sure to choose that change